tests/network/wireguard.pm

```diff
@@ -0,0 +1,85 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
```

Comment:
As we all copy-paste this part from already existing modules, I also tend to forget updating the years. hehe :D